include/range/v3/algorithm.hpp

```diff
@@ -98,6 +99,6 @@
 #include <range/v3/algorithm/aux_/sort_n_with_buffer.hpp>
 #include <range/v3/algorithm/aux_/upper_bound_n.hpp>
 
-#include <range/v3/detail/re_enable_warnings.hpp>
+RANGES_RE_ENABLE_WARNINGS
```

Comment:
How do you reen a warning, so as to determine which warnings are the reenable warnings? ;)

How about simply `ENABLE_WARNINGS`?